src/assets/i18n/en.json

```diff
@@ -0,0 +1,51 @@
+{ "Language" : "English",
+  "HomePage": {
+    "p1": "Start your career",
+    "p2": "And learn with us",
+    "Training": "Training",
```

Comment:
Some keys start with an uppercase letter, some does not. Keep it consistent please